MedicalApp.Domain/Models/Patient.cs

```diff
@@ -12,6 +12,7 @@ public class Patient
         public string FirstName { get; set; }
         public string LastName { get; set; }
         public int Age { get; set; }
-        public Nullable<int> RoomId { get; set; }        
+        //public Nullable<int> RoomId { get; set; }        
+        public Room Room { get; set; }
```

Comment:
Please, provide more informative name, since you will have errors for IntelliSense: class name and variable name are the same.

Reply:
Fixed, I renamed variable as CurrentRoom.